src/styles/index.scss

```diff
@@ -1,3 +1,253 @@
+  font-size: 12px;
+}
+
 body {
   margin: 0;
+  display: grid;
+  grid-template: 60px 1fr/ 1fr;
+}
+
+.header {
+  grid-row: 60px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &-logo {
+    display: block;
+    width: 40px;
+    height: 40px;
+
+    &__img {
+      width: 100%;
+      height: 100%;
+    }
+  }
+
+  &-nav {
+    display: block;
+
+    &__list {
+      display: flex;
+      margin: 0;
+      padding: 0;
+      list-style: none;
+    }
+
+    &__item {
+      margin-left: 20px;
+
+      &:first-child {
+        margin-left: 0;
+      }
+    }
+
+    &__link {
+      position: relative;
+      display: flex;
+      font-weight: 500;
+      color: #060b35;
+      text-decoration: none;
+      text-transform: uppercase;
+      align-items: center;
+      height: 60px;
+      white-space: nowrap;
+
+      &:hover {
+        color: var(--color);
+      }
+    }
+  }
+}
+
+.is-active {
+  color: var(--color);
+
+  &::after {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    height: 4px;
+    border-radius: 2px;
+    background-color: var(--color);
+    content: '';
+  }
+}
+
+.container {
+  display: grid;
+  gap: 48px 46px;
+  margin: 0;
+  justify-content: center;
+  grid-template-columns: repeat(1, 200px);
+}
+
+@media (min-width: 488px) {
+  .container {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .container {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 50px 40px;
+  width: 200px;
+  height: 408px;
+  font-weight: 400;
+
+  &__image {
+    margin-top: 33px;
+
+    &-photo {
+      max-width: 160px;
+      max-height: 134px;
+      object-fit: contain;
+    }
+  }
+
+  &__title {
+    margin-top: 40px;
+    width: 166px;
+    height: 36px;
+    font-size: 12px;
+    font-weight: 500;
+    padding-top: 0;
+    color: #060b35;
+  }
+
+  &__product-code {
+    position: relative;
+    margin-top: 4px;
+    left: -33px;
+    width: 98px;
+    height: 14px;
+
+    &-style {
+      white-space: nowrap;
+      font-size: 10px;
+      line-height: 14px;
+      color: #616070;
+      margin: 0;
+    }
+  }
+
+  &__grade {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 166px;
+    height: 16px;
+    margin-top: 16px;
+
+    &-stars {
+      display: inline-flex;
+      box-sizing: border-box;
+      width: 96px;
+      height: 16px;
+
+      &-list {
+        list-style: none;
+        margin: 0;
+
+        &:nth-child(n + 1) {
+          background-image: url(../images/star-active.svg);
+          width: 16px;
+          height: 16px;
+        }
+
+        &:nth-child(n + 5) {
```

Comment:
The selector '&:nth-child(n + 5)' is incorrect for hiding elements beyond the fourth child. It should be '&:nth-child(n + 6)' to hide elements starting from the sixth child, assuming you want to show the first five.